test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/SortTests.cs

```diff
@@ -26,33 +25,6 @@ public SortTests(HttpIntegrationTestFixture fixture)
         {
         }
 
-        [Fact]
-        public async Task GivenPatients_WhenSearchedWithUnsupportedSortParams_ThenSortIsDroppedFromUrl()
-        {
-            var tag = Guid.NewGuid().ToString();
-            var patients = await CreatePatients(tag);
-
-            await Assert.ThrowsAsync<EqualException>(async () => await ExecuteAndValidateBundle($"Patient?_tag={tag}&_sort=name", false, patients.Cast<Resource>().ToArray()));
-        }
-
-        [Fact]
-        public async Task GivenPatients_WhenSearchedWithUnsupportedSortParamsCode_ThenSortIsDroppedFromUrl()
```

Comment:
Are unsupported sort parameters still dropped from the url?

Reply:
We output the actual sorting used in the query in the URL rather than trying to remove the unsupported parameters.